### PR TITLE
add storage.py & test

### PR DESCRIPTION
**Pull request by sunxiaobiu**
to run the sample:
python storage.py <filename> <your-bucket>

### CODE REVIEW

**Comment by hunkim** [comment]
I don't think this is a solution. You need to *modify* existing code to
make it work with the gs file system. Did you read the hints I provided?

**Comment by sunxiaobiu** [comment]
According to the hints you mentioned before, I have commit the code that change native python IO to tensorflow.python.lib.io.

File: train.py [hunk 1]
```diff
@@ -1,111 +1,115 @@
...
+        return x, y
 
-from utils import TextLoader
-from model import Model
-
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--data_dir', type=str, default='data/tinyshakespeare',
-                       help='data directory containing input.txt')
-    parser.add_argument('--save_dir', type=str, default='save',
```
Comment (hunkim) [inline train.py]: Why did you remove all options?

File: train.py [hunk 2]
```diff
@@ -1,111 +1,115 @@
...
-                            'config.pkl'        : configuration;
-                            'words_vocab.pkl'   : vocabulary definitions;
-                            'checkpoint'        : paths to model file(s) (created by tf).
-                                                  Note: this file contains absolute paths, be careful when moving files around;
-                            'model.ckpt-*'      : file(s) with model definition (created by tf)
-                        """)
-    args = parser.parse_args()
-    train(args)
-
-def train(args):
```
Comment (hunkim) [inline train.py]: Why remove the train?

File: train.py [hunk 3]
```diff
@@ -1,111 +1,115 @@
...
+        data = file_io.read_file_to_string('gs://your-bucket-path/input_file')
+
+        # Optional text cleaning or make them lower case, etc.
+        #data = self.clean_str(data)
+        x_text = data.split()
+
+        self.vocab, self.words = self.build_vocab(x_text)
+        self.vocab_size = len(self.words)
+
+        vocab_file_f = file_io.write_string_to_file('gs://your-bucket-path/vocab_file')
```
Comment (hunkim) [inline train.py]: static bucket name is not good.

**Comment by hunkim** [comment]
Did you test this code? I am not sure this is even working.

**Comment by sunxiaobiu** [comment]
I am so sorry that I haven't test the code because in mainland China, my credit card can not successfully register the google cloud platform account. So I haven't got a avaliable static bucket name to use.
